stdlib/@python2/lib2to3/pytree.pyi

```diff
@@ -20,7 +20,7 @@ class Base:
     children: list[_NL]
     was_changed: bool
     was_checked: bool
-    def __eq__(self, other: Any) -> bool: ...
+    def __eq__(self, other: object) -> bool: ...
```

Comment:
This can return `NotImplemented`. Should the stub reflect that? This applies to several other methods too.

Reply:
I think NotImplemented is generally special-cased in dunders. We don't need to reflect it.

Reply:
`object.__eq__` can return `NotImplemented`, as can the vast majority of `__eq__` and `__ne__` methods for which there are type annotations in typeshed. But the result of `foo = bar` is nearly always a boolean, even if `foo.__eq__(bar)` returns `NotImplemented`, and I assume that's why the standard approach in typeshed is to annotate these methods as returning `bool`, even if that's not strictly accurate.

Reply:
Ah @JelleZijlstra beat me to it